src/app.ts

```diff
@@ -3,7 +3,7 @@ import express from "express";
 import * as nunjucks from "nunjucks";
 import * as path from "path";
 import { APP_NAME } from "./constants";
-import { errorHandler } from "./controllers/error.controller";
+import  errorHandlers from "./controllers/error.controller";
```

Comment:
Not sure functionally it'll make a difference, but all other imports are surrounded by curly braces, we should porobably be consistent

Reply:
just tried it in mine, ide doesn't like it with brackets, I think because it is the default export.

Reply:
fair enough ignore this comment then